.env.example

```diff
@@ -1,2 +1,11 @@
 VITE_GOOGLEOAUTH_CLIENT_ID="YOUR_CLIENT_ID"
-VITE_API_URL=https://www.amiiboapi.com/api/amiibo/
\ No newline at end of file
+VITE_API_URL=https://www.amiiboapi.com/api/amiibo/
+
+# FIREBASE CONFIG
+VITE_FIREBASE_API_KEY='AIzaSyA2K6kdE5ykTG_UILEATf7U0m5onaSIO2Q'
+VITE_FIREBASE_AUTH_DOMAIN='amiibo-atlas-b5484.firebaseapp.com'
+VITE_FIREBASE_PROJECT_ID='amiibo-atlas-b5484'
+VITE_FIREBASE_STORAGE_BUCKET='amiibo-atlas-b5484.appspot.com'
+VITE_FIREBASE_MESSAGING_SENDER_ID='863696608186'
+VITE_FIREBASE_APP_ID='1:863696608186:web:abc0dc56ca4ed4cd375b83'
+VITE_FIREBASE_MEASUREMENT_ID='G-99K69DNER9'
```

Comment:
I've moved the API keys and other details from firebaseConfig.ts to the .env file, so please check the .env.example file!